arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -1208,18 +1236,18 @@
       }
     },
     "CloudToDeviceProperties": {
-      "description": "The Iot Hub Cloud-To-Device messaging properties.",
+      "description": "The IoT hub cloud-to-device messaging properties.",
       "type": "object",
       "properties": {
         "maxDeliveryCount": {
-          "description": "The max delivery count for the device queue. Range : 1-100.",
+          "description": "The max delivery count for cloud-to-device messages in the device queue. See: https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-messaging#cloud-to-device-messages.",
```

Comment:
API rest docs will be generated from these specs. This means the docs would be stale. I am pretty sure updating the description would then be a low hanging fruit for the service team amongst other things to finish.

Our recommendation is: if the default values are contractual meaning they are tied to an api-version or a change in the default value warrants an api-version change then those defaults should be defined in the spec.